soc/SoC_OCV.py

```diff
@@ -161,11 +210,86 @@ def test(self):
 		print(f"soc = 75, ocv = {ocv2}")
 		print(f"soc = 25, ocv = {ocv3}")
 		print(f"soc = 0, ocv = {ocv4}")
+		print('')
+		print("Correct SOC test:")
+		print(f"ocv = {ocv1}, soc = {self.SoCOCV.correct_soc(ocv1, 1)}")
+		print(f"ocv = {ocv2}, soc = {self.SoCOCV.correct_soc(ocv2, 1)}")
+		print(f"ocv = {ocv3}, soc = {self.SoCOCV.correct_soc(ocv3, 1)}")
+		print(f"ocv = {ocv4}, soc = {self.SoCOCV.correct_soc(ocv4, 1)}")
 
 	def test_plot_graph(self):
 		self.SoCOCV.plot_graph()
+	
+	def test_multiple_correct_soc(self, test_file: str):
+		with open(test_file) as f_in:
+			if np.size(self.voltage_array) == 0 and np.size(self.current_array) == 0:
+				self.voltage_array, self.current_array = np.loadtxt(itertools.islice(f_in, 3, None), delimiter='\t', usecols=(1,3), unpack=True)
+			else:
+				temp_voltage, temp_current = np.loadtxt(itertools.islice(f_in, 3, None), delimiter='\t', usecols=(1,3), unpack=True)
+				self.voltage_array = np.concatenate((self.voltage_array, temp_voltage))
+				self.current_array = np.concatenate((self.current_array, temp_current))
+		
+		self.v_ocv = self.voltage_array + (0.085 * self.current_array)
+		print(f"Size of test: {len(self.v_ocv)}")
+
+		for i in range(len(self.v_ocv)):
+			soc = self.SoCOCV.correct_soc(self.v_ocv[i], self.current_array[i])
 
 if __name__ == '__main__':
 	testing = test_SoC_OCV()
```

Comment:
Can we specify what the point of these tests are? The easiest way I think is to throw it into test methods (that will get picked up by pytest. There are docs about this but I think as long as they're in a `tests` directory in a file called `test_<>.py` and each test is defined as `def test_insert_some_descriptive_name():`, you should be OK) with descriptive test method names